ovs-p4rt/ovs_p4rt.cc

```diff
@@ -2596,17 +2627,30 @@ void ConfigIpMacMapTableEntry(struct ip_mac_map_info ip_info,
       ovs_p4rt::GetForwardingPipelineConfig(session.get(), &p4info);
   if (!status.ok()) return;
 
-  if (ip_info.src_ip_addr.ip.v4addr.s_addr != ANY_INADDR &&
-      ip_info.src_ip_addr.ip.v4addr.s_addr != htonl(LOOPBACK_INADDR)) {
+  if (insert_entry) {
+    auto status_or_read_response =
+        GetVmSrcTableEntry(session.get(), ip_info, p4info);
+    if (status_or_read_response.ok()) {
+      goto try_dstip;
+    }
+  }
+  if (ValidIpAddr(ip_info.src_ip_addr.ip.v4addr.s_addr)) {
     status = ConfigSrcIpMacMapTableEntry(session.get(), ip_info, p4info,
                                          insert_entry);
     if (!status.ok()) {
       // TODO: print some log once logging support is added
     }
   }
 
-  if (ip_info.dst_ip_addr.ip.v4addr.s_addr != ANY_INADDR &
-      ip_info.dst_ip_addr.ip.v4addr.s_addr != htonl(LOOPBACK_INADDR)) {
+try_dstip:
```

Comment:
`try_dstip` suggests that you're attempting an alternative to inserting the src ip addr, rather than that you're moving on to other half of the function. I'd suggest something less tentative, such as `do_destip`.